From 1c91186c4897122bb2a173ccf8a72443e48366b3 Mon Sep 17 00:00:00 2001
From: Fabricio Silva <hi@fabricio.dev>
Date: Tue, 16 May 2023 10:25:19 +0100
Subject: [PATCH] remove root path

When searching for a data, it returns the file + path, and searchs twice for the same movie.
Remove the path keep only the file.

---
 src/dataFiles.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dataFiles.ts b/src/dataFiles.ts
index f9b152b..11a107b 100644
--- a/src/dataFiles.ts
+++ b/src/dataFiles.ts
@@ -21,7 +21,7 @@ export function findPotentialNestedRoots(
 				dirent.isDirectory()
 			)
 		);
-		return [root, ...allDescendants];
+		return [...allDescendants];
 	} else if (
 		DATA_EXTENSIONS.includes(extname(root)) &&
 		!root.includes("sample")
